tests/test_batch.py

```diff
@@ -22,9 +22,10 @@
 from asgi_lifespan import LifespanManager
 from fastapi import Request, Response
 from httpx import AsyncClient
+
 from litserve import LitAPI, LitServer
-from tests.conftest import wrap_litserve_start
 from litserve.server import run_batched_loop
+from tests.conftest import wrap_litserve_start
```

Comment:
I would rather reserve `conftest` for fixtures and this (seems to be) general functionality move to another utils module

Reply:
great suggestion @Borda! I will create an issue for a follow up PR. Or in case if you're interested in doing that please feel free go ahead :D